src/components/pages/Login/index.scss

```
@@ -0,0 +1,30 @@
+@import '../../../styles/variables.scss';
+
+html {
+    background-color: $primary-color;
+}
+
+.login_container {
+    display: flex;
+    justify-content: center;
+
+    form {
+        background-color: rebeccapurple;
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        width: 10%;
+    }
+
+    .login_form {
+        width: 70%;
+        background-color: $accents;
```

```suggestion
        background-color: $accents-color;
```

Me parece que deberíamos especificar que accents en un color